backend/cmd/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
Is this file required?

Reply:
yes, this is the way to run the server, to get coverage for any kind of FT or automation testing

Reply:
Ok. I was confused as this code was not specific to SSE testing but generic